docs/source/introduction.rst

```diff
@@ -1,62 +1,64 @@
 .. _introduction:
 
 VulnerableCode Overview
-========================
+=======================
 
 *VulnerableCode* provides an open database of software packages that are affected
-by known security vulnerabilities aka. *"vulnerable packages"*.
+by known security vulnerabilities aka *"vulnerable packages"*.
 
 VulnerableCode is also a free and open source software (FOSS) project that
 provides the tools to build this open database. The tools handle collecting,
 aggregating and correlating these vulnerabilities and relating them to a correct
 package version. Our project also supports a public cloud instance of this
-database - VulnerableCode.io.
+database - `VulnerableCode.io <https://public.vulnerablecode.io/>`__.
 
 
 What can I do with VulnerableCode?
-------------------------------------
+----------------------------------
 
 **For security researchers and software developers, VulnerableCode offers a web
-UI and a JSON API to efficient find if the FOSS packages and dependencies that
-you use may be affected by known vulnerabilities and which version of a package
-you should upgrade to to fix this issue.**
+UI and a JSON API to efficiently find if the FOSS packages and dependencies that
+you use are affected by known vulnerabilities and to determine whether a later package version
+fixes those vulnerabilities.**
 
 
 - With the web UI, you can search by package using Package URLs or search by
-  vulnerability like by CVE. From there you can navigate to the package
+  vulnerability, e.g., by CVE. From there you can navigate to the package
   vulnerabilities and to the vulnerable packages.
 
-- With the JSON API, you can perform package queries using Package URLs or query
-  by vulnerability id. You can also query by CPEs and vulnerability aliases.
+- With the JSON API, you can perform package queries using Package URLs (`purl
+  <https://github.com/package-url/purl-spec>`__) or query
+  by vulnerability id. You can also query by CPEs and other vulnerability aliases.
```

Comment:
I think it should be VCID and not vulnerability id.
```suggestion
  by VCID. You can also query by CPEs and other vulnerability aliases.
```

Reply:
Thank you @TG1999 👍  -- I changed that passage to `. . . or query by vulnerability id ("VCID").`